extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -199,8 +205,13 @@ export abstract class AzureAuth implements vscode.Disposable {
 				if (!baseToken) {
 					return undefined;
 				}
+				try {
+					await this.refreshAccessToken(account.key, baseToken.refreshToken, tenant, resource);
+				} catch (ex) {
```

Comment:
Hm, don't want to log/display the error anywhere?

Reply:
The potential places it could try exceptions is logged before this. I think this is fine as anything else would just create too much spam when it fails for the proper reasons.

Reply:
Yeah. I'm thinking maybe we should make a more concerted effort to have extensions actually set up a debug log and/or trace level or something. Would be nice not to log every thing to console but sometimes we want to be able to for debugging purposes.